data/deep missions.txt

```diff
@@ -0,0 +1,2317 @@
+			names "civilian"
+			variant
+				"Freighter" 3
+				"Mule" 2
+	npc
+		government "Merchant"
+		personality timid
+		fleet
+			fighters "civilian"
+			variant
+				"Dagger" 2
+			variant
+				"Boxwing" 2
+			variant
+				"Dagger"
+				"Boxwing"
+	npc
+		government "Pirate"
+		personality nemesis staying
+		system "Tania Australis"
+		fleet "Large Northern Pirates"
+	npc
+		government "Pirate"
+		personality nemesis staying
+		system "Muphrid"
+		fleet "Large Northern Pirates"
+	npc
+		government "Pirate"
+		personality nemesis staying
+		system "Ascella"
+		fleet "Small Northern Pirates"
+	npc
+		government "Pirate"
+		personality nemesis
+		system "Sirius"
+		fleet "Large Northern Pirates"
+	npc
+		government "Pirate"
+		personality nemesis
+		system "Seginus"
+		fleet "Small Northern Pirates"
+	npc
+		government "Pirate"
+		personality nemesis staying
+		system destination
+		fleet "Large Northern Pirates"
+	
+	on stopover
+		dialog `You and the freighters are directed to park in a private hangar owned by the Tarazed Corporation. While watching the cargo crates get loaded on to the freighters, you notice that the crates have labels on them which read "To be opened by addressee only" in large red letters.`
+	on complete
+		"deep convoy" ++
+		payment 650000
+		conversation
+			`When you return to <planet> with the freighters, you are greeted by a Deep Security officer who hands you <payment> and thanks you for accepting the job on such a short notice. "We'll make sure to contact you in the future should we need your help. If you want to help us further at the moment, I suggest you check the job boards in the Deep for mystery delivery and retrieval jobs."`
+
+
+
+mission "Deep: Kraz Convoy"
+	minor
+	name `Pickup from Kraz`
+	description `Escort a convoy of freighters to <stopovers> where they will pick up materials needed for scientific experiments. Return the convoy to <destination> by <date> for payment (<payment>).`
+	source
+		attributes "deep"
+		near "Gomeisa" 2
+	stopover "Rust"
+	deadline
+	to offer
+		"combat rating" > 80
+		not "Deep: Mystery Cubes 4: done"
+		or
+			random < 20
+			and
+				"deep convoy" == 1
+				random < 30
+			and
+				"deep convoy" == 2
+				random < 40
+	
+	on offer
+		conversation
+			`A Deep Security officer approaches you when you enter the spaceport and asks if you would be able to do a job for the Deep. "We're short on Deep Security ships at the moment, but we need three Behemoths escorted to <stopovers>. After they pick up the cargo, bring them back here by <day>."`
+			choice
+				`	"Sure, I can do that."`
+				`	"Sorry, but I won't be traveling in that direction any time soon."`
+					decline
+			
+			`	"Much appreciated, Captain. And for your well being as well as that of the cargo's, you must know that there may be pirates waiting in Algieba, Cor Caroli, and Wei. Behemoths are tough ships so you should be able to travel through those systems and not slow yourself down by taking alternative routes."`
```

Comment:
I recommend updating this line to:
```"Much appreciated, Captain. For your well being as well as that of the cargo, you must know that there may be pirates waiting in Algieba, Cor Caroli, and Wei. Behemoths are tough ships, so you should be able to travel through those systems and not slow yourself down by taking alternative routes."```

The "And" is unnecessary, and "cargo's" is grammatically incorrect. I also added a comma to break up the last sentence.